Makefile

```diff
@@ -54,7 +54,8 @@ doc-deps: ## Build the documentation for the local package and all dependencies.
 
 .PHONY: gen-rpc-doc
 gen-rpc-doc:  ## Generate rpc documentation
-	rm -f target/doc/ckb_rpc/module/trait.*.html
+	rm -f ${CARGO_TARGET_DIR}/doc/ckb_rpc/module/trait.*.html
+	[ ! -h "target" ] && ln -nsf "${CARGO_TARGET_DIR}" "target"
```

Comment:
Why this step is required?

Reply:
Too avoid to update `test.sh` and `rpc.py`, :dog: